tests/FsUnit.Xunit.Test/equalTests.fs

```diff
@@ -30,6 +30,14 @@ type ``equal Tests`` ()=
     [<Fact>] member test.
      ``value type should fail to not equal equivalent value`` ()=
         1 |> should equal 1
+        
+    [<Fact>] member test.
+     ``collection type should equal collection`` ()=
+        [1..10] |> should equal [1..10]
+
+    [<Fact>] member test.
+     ``collection type should not equal equivalent if is not in same order`` ()=
+        [1;2;3] |> should not' (equal [3;2;1])
```

Comment:
@CaptnCodr I mean here. There is no tests that call/uses `equivalent` operator for xUnit.

Reply:
I didn't implement this for xUnit. Because I impemented `CollectionAssert.AreEquivalent` from MsTest in `FsUnit.fs`. In a previous [comment](https://github.com/fsprojects/FsUnit/issues/133#issuecomment-542027587) you said "we can leave it not implemented".